.github/label-actions.yml

```diff
@@ -5,13 +5,90 @@
+        - We or the contributor doesn't have the time or equipment necessary to test it or fix it up
+        - Sometimes the implementation isn't quite right and a different approach is necessary
+        
+        We would love to land this pull request when it's ready. If you have a chance to address the comments, we'd be happy to reopen and discuss merging this contribution into the framework! :tada:
+        
     needs-docs:
       comment: |
-        Thanks for your pull request, before this can be merged - corresponding documentation for your module is required:
+        Thanks for your pull request! Before this can be merged, we need the following documentation for your module:
+
+        - [Writing Module Documentation](https://github.com/rapid7/metasploit-framework/wiki/Writing-Module-Documentation)
+        - [Template](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/module_doc_template.md)
+        - [Examples](https://github.com/rapid7/metasploit-framework/tree/master/documentation/modules)
+
+    needs-linting:
+      comment: |
+        Thanks for your pull request! We currently use Rubocop and msftidy to ensure the quality of our code. This can be ran from the root directory of Metasploit:
+        
+        ```
+        rubocop <directory or file>
+        tools/dev/msftidy.rb <directory or file>
+        ```
+        
+        It's also possible to automate some of these changes with the `-a` flag:
+        
+        ```
+        rubocop -a <directory or file>
+        ```
+        
+        After making these changes, push to your branch again once all issues have been resolved.
+
+    needs-unique-branch:
+      close: true    
+      comment: |
+        Thanks for your pull request! We require for all contributed code to come from a **from a unique branch** in your repository before it can be merged.
+
+        Please create a new branch in your fork of framework and resubmit this from that branch.
+
+        If you are using Git on the command line that may look like:
+
+        ```
+        # Checkout the master branch
+        git checkout master
+
+        # Create a new branch for your feature
+        git checkout -b <BRANCH_NAME>
+
+        # Add your new files
+        git add modules/my-cool-new-module
+
+        # Commit your changes with a relevant message
+        git commit
+
+        # Push your changes to GitHub
+        git push origin <BRANCH_NAME>
+
+        # Now browse to the following URL and create your pull request!
+        # - https://github.com/rapid7/metasploit-framework/pulls        
+        ```
+
+        This helps protect the process, ensure users are aware of commits on the branch being considered for merge, allows for a location for more commits to be offered without mingling with other contributor changes and allows contributors to make progress while a PR is still being reviewed.
```

Comment:
I've taken this from:

- https://github.com/rapid7/metasploit-framework/pull/11200 
- https://github.com/rapid7/metasploit-framework/wiki/So-Your-PR-was-closed#moved-to-the-attic

But I'm not quite sure I understand what it means, is there a more concise way to write it maybe? 🤔
Or it might make sense to update the wiki page with more details, and have the bot just link to the article.

Reply:
Related issue #11224 is referenced as documentation. It was meant to be an issue for someone-who-isn't-me to write some documentation. Ideally, the documentation could then be reference here.